vendor/parser-standard/jane_syntax_parsing.mli

```diff
@@ -98,7 +98,7 @@ module Feature : sig
     | Builtin
 
   (** The component of an attribute or extension name that identifies the
-      feature. This is third component.
+      feature. This is the third component.
```

Comment:
I assume these parser-standard changes are just copied over from your branch -- let me know if that's not the case.

Reply:
That's correct, those files are literally `cp`ed over.